tests/robot-tests/tests/general_public/miscellaneous.robot

```diff
@@ -15,31 +15,33 @@ Verify public page loads
 
 Verify can accept cookie banner
     [Tags]  HappyPath   NotAgainstLocal
-    user checks page contains  GOV.UK uses cookies to make the site simpler.
+    user checks page contains  We use some essential cookies to make this service work.
 
     cookie should not exist   ees_banner_seen
     cookie should not exist   ees_disable_google_analytics
 
-    user clicks element  xpath://button[text()="Accept Cookies"]
+    user clicks element  xpath://button[text()="Accept analytics cookies"]
 
     cookie should have value  ees_banner_seen   true
     cookie should have value  ees_disable_google_analytics   false
+    user clicks button  Hide this message
```

Comment:
Worth adding a quick test to ensure that clicking Hide this message does actually hide the cookie banner